internal/pkg/template/templates/environment/cf.yml

```
@@ -557,6 +557,63 @@ Resources:
 {{include "lambdas" . | indent 2}}
 {{include "custom-resources" . | indent 2}}
 {{- end}}
+{{- if .VPCConfig.Flowlogs }}
+  Flowlog:
```

nit (matches https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-flowlog.html)
```suggestion
  FlowLog:
```